src/plots/plots.js

```diff
@@ -809,6 +809,16 @@ plots.purge = function(gd) {
     // remove modebar
     if(fullLayout._modeBar) fullLayout._modeBar.destroy();
 
+    if(fullLayout._plots) {
```

Comment:
@etpinard I was chasing a point pick regression but back on this now. The root call is a `Plotly.newPlot`. It does call `supplyDefaults` and `cleanPlot`. But [`oldFullData`](https://github.com/plotly/plotly.js/blob/46bbbaebfd70ed28196df9c4993a1ded5292bebf/src/plots/plots.js#L563) is `[]` empty so it doesn't even reach the point of testing if there is a `_modue.clean` metod. I'm carrying on looking, based on your hint, but wanted to mention it in case it rings a bell.